src/android/ReflectiveCordovaPlugin.java

```diff
@@ -48,6 +48,11 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
         return false;
     }
 
+    protected void handleUncaughtExceptions(final Throwable e, final Method method, final Object[] methodArgs, final CallbackContext callbackContext) {
```

Comment:
`final` keyword for each argument is unnecessary here

Reply:
No worries 👍